docs/security/trust/content_trust.md

```diff
@@ -14,14 +14,10 @@ weight=-1
 When transferring data among networked systems, *trust* is a central concern. In
 particular, when communicating over an untrusted medium such as the internet, it
 is critical to ensure the integrity and publisher of all the data a system
-operates on. You use Docker to push and pull images (data) to a registry. Content trust
+operates on. You use Docker to push and pull images (data) to a public or private registry. Content trust
 gives you the ability to both verify the integrity and the publisher of all the
```

Comment:
I know this wasn't part of your change, but the next sentence is phrased a little oddly.  Maybe "verify both the integrity and the publisher"...?